boards/nucleo-l452re/doc.txt

```diff
@@ -8,6 +8,29 @@
 The Nucleo-L452RE is a board from ST's Nucleo family supporting ARM Cortex-M4
 STM32L452RE microcontroller with 160KiB or RAM and 512KiB of Flash.
```

Comment:
```suggestion
STM32L452RE microcontroller with 160KiB of RAM and 512KiB of Flash.
```

maybe, while at it, you can fix this typo

Reply:
Fixed!

Thanks for spotting this typo.